admin_manual/installation/harden_server.rst

```diff
@@ -240,11 +240,13 @@ Some Nextcloud functionality requires connecting to remote servers. Depending on
 your server setup those are possible connections:
 
 - www.nextcloud.com, www.startpage.com, www.eff.org, www.edri.org for checking the internet connection
-- apps.nextcloud.com for the available apps
-- updates.nextcloud.com for Nextcloud updates
-- lookup.nextcloud.com For updating and lookup in the federated sharing addressbook
-- push-notifications.nextcloud.com for sending push notifications to mobile clients
-- surveyserver.nextcloud.com if the admin has agreed to share anonymized data
+- cloud.nextcloud.com (https) for validating the enterprise subscription
+- apps.nextcloud.com (https) for available apps and their updates 
+- github.com (https) for downloading Nextcloud standard apps
```

Comment:
Can we move this to the end, so it's more visible (since it's not our server)

Reply:
Also pushfeed.nextcloud.com (https) could be listed:

https://github.com/nextcloud/nextcloud_announcements/blob/master/lib/Cron/Crawler.php#L38C35-L38C57